addOns/reports/src/main/java/org/zaproxy/addon/reports/automation/ReportJob.java

```diff
@@ -101,6 +142,65 @@ public void runJob(
         }
     }
 
+    private int riskStringToInt(String str, AutomationProgress progress) {
+        switch (str.toLowerCase()) {
+            case "high":
+                return Alert.RISK_HIGH;
+            case "medium":
+                return Alert.RISK_MEDIUM;
+            case "low":
+                return Alert.RISK_LOW;
+            case "info":
+                return Alert.RISK_INFO;
+            case "information":
+                return Alert.RISK_INFO;
+        }
+        progress.warn(
+                Constant.messages.getString(
+                        "reports.automation.error.badrisk", this.getName(), str));
+        return -2;
+    }
+
+    private int confidenceStringToInt(String str, AutomationProgress progress) {
+        switch (str.toLowerCase()) {
+            case "high":
+                return Alert.CONFIDENCE_HIGH;
+            case "medium":
+                return Alert.CONFIDENCE_MEDIUM;
+            case "low":
+                return Alert.CONFIDENCE_LOW;
+            case "falsepositive":
+                return Alert.CONFIDENCE_FALSE_POSITIVE;
+        }
+        progress.warn(
+                Constant.messages.getString(
+                        "reports.automation.error.badrisk", this.getName(), str));
+        return -2;
+    }
+
+    private List<String> getJobDataList(
+            LinkedHashMap<?, ?> jobData, String key, AutomationProgress progress) {
+        Object o = jobData.get(key);
+        if (o == null) {
+            return null;
```

Comment:
`null` does not seem to have a special meaning, in which case it's better to return always a list (avoids the need to check for nulls elsewhere).